src/components/dialogs/network-modifications/vsc/converter-station/converter-station-utils.tsx

```diff
@@ -176,7 +176,7 @@ export function getVscConverterStationSchema(id: string) {
 export function getVscConverterStationModificationSchema(id: string) {
     return {
         [id]: yup.object().shape({
-            [CONVERTER_STATION_ID]: yup.string().nullable(),
+            [CONVERTER_STATION_ID]: yup.string(),//.nullable(),
```

Comment:
should be deleted if not needed, also we can required if it is the case

Reply:
done